docs/API.md

```diff
@@ -715,29 +721,23 @@ Example response:
 
 An object with the following properties:
 
-* `"offset"`: An integer, optional. (default: 0).
-* `"limit"`: An integer, optional. (default: 200).
+* `"offset"`: An integer, optional. (default: 0). Position of the first returned element from the database returned list.  
+* `"limit"`: An integer, optional. (default: 200). Number of element returned from the *offset* element.
 * `"frontend_params"`: As described below.
 
-The value of `"frontend_params"` is an object in turn, with the
-keys `"domain"` and `"nameservers"`. `"domain"` and `"nameservers"`
-will be used to look up all tests for the given domain, separated
-according to if they were started with a `"nameservers"` parameter or
-not.
-
->
-> TODO: Do we have an SQL injection opportunity here?
->
-> TODO: Describe the remaining keys in the example
->
-> TODO: Describe the purpose of `"offset"` and `"limit"`
->
-> TODO: Is the `"nameservers"` value a boolean in disguise?
->
-> TODO: The description of `"frontend_params"` is clearly not up to date. Can it
-> be described in a better way?
->
-
+* `"client_id"`: A free-form string, optional.
+* `"profile"`: A *profile name*, optional.
+* `"client_version"`: A free-form string, optional.
+* `"nameservers"`: A boolean in order to return either "regular" (false) or "undelegated" (true).
+* `"ds_info"`: A list of *DS info* objects, optional.
+* `"advanced"`: **Deprecated**. A boolean, optional.
+* `"ipv6"`: A boolean, optional. (default: `false`)
+* `"ipv4"`: A boolean, optional. (default: `false`)
+* `"config"`: A string, optional. The name of a *config profile*.
+* `"user_ip"`: An IP, optional.
+* `"user_location_info"`: An *location* object, optional.
+* `"priority"`: A *priorty*, optional.
+* `"queue"`: A *queue*, optional.
```

Comment:
These fields have no relevance for get_test_history as far as I can see.

Reply:
@matsduf i agree